.github/workflows/ci.yaml

```diff
@@ -15,15 +15,14 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [ 16.x ]
+        node-version: [ 20.x ]
         os: [ ubuntu-latest, windows-latest, macos-latest ]
 
     steps:
       - uses: actions/checkout@v3
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v3
         with:
-          cache: yarn
```

Comment:
github runners already have some node installed, so running corepack enable before setup-node fixed things. Alternative is to run:
1. actions/setup-node,
2. corepack/enable
3. actions/cache

Reply:
here's a run when I tried putting `corepack enable` before `setup-node` https://github.com/ChainSafe/eslint-config/actions/runs/7461021466/job/20300293421

I'll try out the alternative